requirements/base.in

```diff
@@ -14,3 +14,6 @@ django-simple-history                   # History for Django models
 django-object-actions                   # Object actions in Django admin
 edx-rest-api-client                     # For accessing the Enrollment API (and possibly other edX APIs)
 django-config-models                    # Provides ConfigurationModel abstract base class
+requests                                # Required for SAPSuccessFactorsAPIClient
+simplejson                              # Provides JSONDecodeError
```

Comment:
I began writing a very long thing about all the different ways requests might produce JSON errors due to the way it's implemented, but installing and using simplejson should work in most cases (as I don't think we're ever migrating to Python 3.2), so I'm OK with this. The alternative is pretty ugly, since the built-in `json` module that requests uses when `simplejson` isn't installed changed its exception API completely between python2 and python3, and six doesn't have a shim for that difference.